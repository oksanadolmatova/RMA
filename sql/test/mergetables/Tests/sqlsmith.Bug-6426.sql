select  
  sample_4.x as c0, 
  sample_0.location as c1
from 
  sys.netcdf_files as sample_0
      right join tmp.idxs as sample_1
          right join sys.types as sample_2
          on (sample_1.table_id = sample_2.id )
        inner join sys.s as sample_4
          inner join tmp.idxs as ref_0
          on (false)
        on (sample_2.digits = ref_0.id )
      on (sample_0.file_id = sample_4.x )
    left join sys.netcdf_files as sample_6
          inner join sys.netcdf_dims as ref_1
            inner join sys.s1 as ref_2
            on (false)
          on (sample_6.file_id = ref_1.dim_id )
        right join bam.rg as sample_7
        on (ref_2.x = sample_7.pi )
      left join sys.netcdf_dims as ref_3
            right join tmp._columns as ref_4
            on (ref_3.length is NULL)
          right join sys.triggers as sample_8
          on (ref_3.name = sample_8.name )
        left join sys.systemfunctions as ref_5
        on (ref_3.dim_id is not NULL)
      on ((false) 
          or (true))
    on (sample_0.file_id = sample_8.id )
where case when true then ref_0.name else ref_0.name end
     is NULL;
