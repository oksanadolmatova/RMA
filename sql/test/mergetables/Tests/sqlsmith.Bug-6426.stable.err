stderr of test 'sqlsmith.Bug-6426` in directory 'sql/test/mergetables` itself:


# 08:18:22 >  
# 08:18:22 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34984" "--set" "mapi_usock=/var/tmp/mtest-22726/.s.monetdb.34984" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_mergetables"
# 08:18:22 >  

# builtin opt 	gdk_dbpath = /home/niels/scratch/rc-monetdb/Linux-x86_64/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 34984
# cmdline opt 	mapi_usock = /var/tmp/mtest-22726/.s.monetdb.34984
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_mergetables
# cmdline opt 	gdk_debug = 536870922

# 08:18:22 >  
# 08:18:22 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-22726" "--port=34984"
# 08:18:22 >  


# 08:18:22 >  
# 08:18:22 >  "Done."
# 08:18:22 >  

