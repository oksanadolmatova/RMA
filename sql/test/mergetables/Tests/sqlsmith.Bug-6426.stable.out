stdout of test 'sqlsmith.Bug-6426` in directory 'sql/test/mergetables` itself:


# 08:18:22 >  
# 08:18:22 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=34984" "--set" "mapi_usock=/var/tmp/mtest-22726/.s.monetdb.34984" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_mergetables"
# 08:18:22 >  

# MonetDB 5 server v11.27.6
# This is an unreleased version
# Serving database 'mTests_sql_test_mergetables', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 7.330 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:34984/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-22726/.s.monetdb.34984
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 08:18:22 >  
# 08:18:22 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-22726" "--port=34984"
# 08:18:22 >  

#select  
#  sample_4.x as c0, 
#  sample_0.location as c1
#from 
#  sys.netcdf_files as sample_0
#      right join tmp.idxs as sample_1
#          right join sys.types as sample_2
#          on (sample_1.table_id = sample_2.id )
#        inner join sys.s as sample_4
#          inner join tmp.idxs as ref_0
#          on (false)
#        on (sample_2.digits = ref_0.id )
#      on (sample_0.file_id = sample_4.x )
#    left join sys.netcdf_files as sample_6
#          inner join sys.netcdf_dims as ref_1
#            inner join sys.s1 as ref_2
% .L2,	.L4 # table_name
% c0,	c1 # name
% int,	char # type
% 1,	256 # length

# 08:18:22 >  
# 08:18:22 >  "Done."
# 08:18:22 >  

